alfred/info.plist

```diff
@@ -101,31 +103,55 @@
 	</array>
 	<key>readme</key>
 	<string>Type `ns` and two station names to see train schedule between them.
-Make sure to get API key from https://apiportal.ns.nl/ and set it as workflow environment variable NS_APIKEY</string>
+
+![](images/screenshot.png)
```

Comment:
nice improvement of the readme, however directory names are going to be ignored when building the final `.alfredworkflow` file, so this path would be invalid.

I would also rather add the path to the readme image to the `ALFRED_PACKAGE_FILES` variable in the Makefile rather than duplicate the image file

Reply:
Done.